pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
@tanyaparker I would rather just leave them as-is and reduce migration complexity.